server/src/tenant/schema-builder/utils/map-column-type-to-filter-type.util.ts

```diff
@@ -0,0 +1,39 @@
+import { GraphQLBoolean } from 'graphql';
+
+import { FieldMetadata } from 'src/metadata/field-metadata/field-metadata.entity';
+import { UUIDFilterType } from 'src/tenant/schema-builder/graphql-types/input/uuid-filter.type';
+import { StringFilterType } from 'src/tenant/schema-builder/graphql-types/input/string-filter.type';
+import { DateFilterType } from 'src/tenant/schema-builder/graphql-types/input/date-filter.type';
+import { IntFilter } from 'src/tenant/schema-builder/graphql-types/input/int-filter.type';
+import { UrlFilterType } from 'src/tenant/schema-builder/graphql-types/input/url-filter.type';
+import { MoneyFilterType } from 'src/tenant/schema-builder/graphql-types/input/money-filter.type';
+
+/**
+ * Map the column type from field-metadata to its corresponding filter type.
+ * @param columnType Type of the column in the database.
+ */
+export const mapColumnTypeToFilterType = (column: FieldMetadata) => {
+  switch (column.type) {
+    case 'uuid':
+      return UUIDFilterType;
+    case 'text':
```

Comment:
To discuss: It seems we are doing this kind of check in many places now, we should definitely see if we can have something cleaner to implement new types without editing many files. (An interface class with a class for each type where we need to implement some methods for example?)

Reply:
Yes it's planed but I'm waiting for now until everything is working smoothly, and I'll make something more clean :)